src/index.js

```diff
@@ -0,0 +1,94 @@
+temp = 70
+
+const updateTemp = temp => {
+    const tempFarenheitContainer= document.getElementById("tempFarenheit");
+    tempFarenheitContainer.textContent = temp;
+}
+const increaseTemp = () => {
+    temp += 1;
+    updateTemp(temp);
+};
+const decreaseTemp = () => {
+    temp -=1;
+    updateTemp(temp);
+};
+const updateTempColor = (updatedTemp)=>{
+    const tempFarenheitContainer= document.getElementById("tempFarenheit");
+    let tempColor = "black";
+    if (updatedTemp >= 80){
+        tempColor = "red"
+    }   else if (updatedTemp >=70){
+            tempColor = "orange"
+    }   else if (updatedTemp >=60){
+            tempColor = "yellow"
+    }   else if (updatedTemp >=50){
+            tempColor = "green"
+    }   else {
+            tempColor = "blue"
+    }
+    tempFarenheitContainer.textContent = tempFarenheit
```

Comment:
We don't want to set the `textContent` of the temperature display here (that's already being done in `updateTemp `). Here, we want to use the class we calculated to update the style on the container. If there were classes in the css to set the color using these calculated names, such as the following (though it's a slight anti-pattern to describe the style with its visual effect rather than a behavior)

```css
.yellow {
  color: yellow;
}
```

then here, we could set the class name with
```js
tempFarenheitContainer.className = tempColor;
```